docs/dev.rst

```diff
@@ -1,20 +1,513 @@
+
+The above runs the ``dev_org`` flow against the org currently defined as ``dev`` in CumulusCI.
+
+You can now open up your new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+
+Setting the Capture State
+^^^^^^^^^^^^^^^^^^^^^^^^^
+When you are ready to start making changes that you want to capture in an org, start by creating a snapshot.
+This tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
+
+.. note::
+
+    If you have just set up a scratch org by running the ``dev_org``, ``dev_org_namespaced``,
+    ``qa_org``, ``regression_org``, ``install_beta`` or ``install_prod`` flows, you do not need to run ``snapshot_changes``.
+    These flows all run ``snapshot_changes`` as their last step.
+
+.. code-block:: console
+
+    $ cci task run snapshot_changes --org dev
+
+This will allow the ``list_changes`` and ``retrieve_changes`` tasks to detect any new metadata but ignore any prior changes.
+A number of the standard CumulusCI flows, include the ``snapshot_changes`` as the final step.
+
+Check that the snapshot was created successfully:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+This should list no changes to the org.
+
+
+
+List Changes
+------------
+You can check what components you have changed in a target org with the ``list_changes`` command:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+.. note::
+    
+    This functionality relies on Salesforce's source tracking feature, so it is only available in scratch orgs.
+
+You can also include/exclude components from the list using the include/exclude options:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --include "test.*,another_regex" --exclude "something_to_exclude"
+
+The ``include`` and ``exclude`` patterns are matched against both the metadata type and name of the component.
+
+You can also include all changed components of specific types:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --types "CustomObject,CustomField"
+
+
+
+Retrieve Changes
 ----------------
+The ``retrieve_changes`` task supports both ``sfdx`` and ``mdapi`` formatted source code. 
+It also utilizes the `SourceMember <https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/tooling_api_objects_sourcemember.htm>`_ sObject to detect what has changed in an org,
+but allows you to be more selective regarding which components to retrieve when compared to the ``dx_pull`` task. 
+Manual tracking of component versions also allows for the possibility of retrieving some changes into one directory,
+and then running the task again to retrieve other changes into a different directory.
+ 
+.. note::
+
+    CumulusCI has multiple tasks for retrieving Metadata from an org environment.
+    For a comprehensive list, see the list of :ref:`retrieve changes` tasks in the cheat sheet.
```

Comment:
I don't think the cheat sheet is a good place for a comprehensive list of anything. I would list them briefly here: In addition to retrieve_changes, CumulusCI provides some other tasks for use in specific scenarios: retrieve_packaged to retrieve an entire unmanaged package, etc... and link each task name to its entry in the task reference.